openpype/hosts/flame/api/pipeline.py

```diff
@@ -68,14 +70,33 @@ def uninstall():
     log.info("OpenPype Flame host uninstalled ...")
 
 
-def containerise(tl_segment,
+def containerise(flame_clip,
                  name,
                  namespace,
                  context,
                  loader=None,
                  data=None):
-    # TODO: containerise
-    pass
+
+    data_imprint = OrderedDict({
```

Comment:
Is `OrderedDict` really needed? BTW you are creating ordered dictionary from standard dictionary so e.g. in Python 2 it would not be ordered as is in code.

Reply:
yeah,you are right, this is just inherited way from nuke where it makes sense as the order in knobs is nicer..